src/routes/+page.svelte

```diff
@@ -2,24 +2,107 @@
 	import { get } from 'svelte/store';
 	import Login from '../components/Login.svelte';
 	import Feed from '../components/Feed.svelte';
-	import { serverURL, token } from '$lib/Store';
-	import ChangeFeedTypeButton from '../components/ChangeFeedTypeButton.svelte';
+	import { token } from '$lib/Store';
 	import { onMount } from 'svelte';
-	import { getToastStore } from '@skeletonlabs/skeleton';
-	import { fetchPosts } from '$lib/FeedFunctions';
+	import { RadioGroup, RadioItem, getToastStore } from '@skeletonlabs/skeleton';
+	import type { Feed as FeedStructure } from '$lib/types/Feed';
+	import { fetchPosts } from '../lib/FeedPosts';
+	import { t } from '../i18n';
 
-	const toastStore = getToastStore();
+	let hasMorePosts = true;
+	let maxPostCounter = 0;
+	let slotLimit = 2;
+	let feedType = 'global';
+	let feedData: FeedStructure = {
+		records: [],
+		pagination: {
+			lastPostId: '',
+			limit: slotLimit,
+			records: 0
+		}
+	};
+	let value: boolean = false; // FeedType: true = personal, false = global
 
+	const toastStore = getToastStore();
 	const loginToken = get(token);
+
 	onMount(async () => {
-		const loginToken = get(token);
-		const url = get(serverURL);
+		let result;
```

Comment:
wenn du result außerhalb der Funktion definierst, dann sparst du dir das kopieren der Werte in Zeile 41-43. du kannst dann in der ganzen Datei auf die werte zugreifen. Ist aber eigentlich egal

Reply:
Wenn du es so lässt würde ich es jeweils an den Stellen, wo du das result bekommst (32 & 45) einfach const machen und dann die let Initialisierung weg machen (30)